src/internal/svg/coinbasePaySvg.tsx

```diff
@@ -0,0 +1,21 @@
+import { fill } from '../../styles/theme';
+
+export const coinbasePaySvg = (
+  <svg
+    role="img"
+    aria-label="ock-coinbasePaySvg"
+    width="20"
+    height="20"
```

Comment:
To match the other SVG:

```suggestion
    width="100%"
    height="100%"
```

This allows the SVGs to be more flexible, relying on the parent component for sizing